lrspline/raw.pyx

```diff
@@ -421,11 +438,29 @@ cdef class LRSplineObject:
     def setMaxAspectRatio(self, double r, bool aposteriori=True):
         self.w.setMaxAspectRatio(r, aposteriori)
 
-
 cdef class LRSurface(LRSplineObject):
 
-    def __cinit__(self, w=None):
-        self.w = new LRSplineSurface_()
+    def __cinit__(self, n1=None, n2=None, order_u=None, order_v=None, knot1=None, knot2=None, coef=None, dim=2):
+        cdef array.array knot1_arr;
+        cdef array.array knot2_arr;
+        cdef array.array coef_arr;
+        if knot1 is not None and knot2 is not None and coef is not None:
+            knot1_arr = array.array('d', knot1)
```

Comment:
I didn't try it but I guess it's also possible to do something like this:

```cpython
cdef vector[double] knot1_arr
knot1_arr = knot1
```

and then later, to get a pointer, do

```cpython
&knot1_arr[0]
```

But this is fine too.

Reply:
This was picked up from a stackoverflow post which I can't seem to relocate. I don't know the exact reason it works, but it does so I dare not touch it. If you want to refactor it at a later point you are free to do this in a separate PR, but for now I think we'll keep it like this.